native/cocos/renderer/gfx-base/GFXDef-common.h

```diff
@@ -1396,7 +1400,7 @@ struct ALIGNAS(8) BufferBarrierInfo {
 
     uint32_t offset{0};
     uint32_t size{0};
-
+    uint32_t _padding{0};
```

Comment:
I forgot the padding changed member initialization order. It can be problematic.

Reply:
Do we depend on the order？